index.html

```diff
@@ -0,0 +1,134 @@
+      </svg>
+    </div>
+    <div class="shape-6">
+      <svg width="24" height="28" viewBox="0 0 24 28" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path d="M-11.8095 0.641908C-13.8144 5.33769 -13.8718 10.6376 -11.969 15.3757C-10.0663 20.1137 -6.35924 23.9019 -1.66346 25.9068C3.03231 27.9117 8.3322 27.969 13.0703 26.0663C17.8084 24.1635 21.5965 20.4565 23.6014 15.7607L-11.8095 0.641908Z" fill="#6070FF"/>
+      </svg>
+    </div>
+    <div class="shape-7">
+      <svg width="9" height="44" viewBox="0 0 9 44" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <rect x="-34" y="37.4868" width="51.9898" height="9.30073" transform="rotate(-46.1415 -34 37.4868)" fill="#EBEBFF"/>
+      </svg>
+    </div>
+    <div class="shape-8">
+      <svg width="25" height="13" viewBox="0 0 25 13" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path d="M0.496961 -0.000453654C0.278167 3.19352 1.33714 6.3436 3.44091 8.75679C5.54468 11.17 8.52093 12.6486 11.7149 12.8674C14.9089 13.0862 18.059 12.0272 20.4722 9.92347C22.8853 7.8197 24.364 4.84345 24.5828 1.64948L0.496961 -0.000453654Z" fill="#6070FF"/>
+      </svg>
+    </div>
+    <div class="shape-9">
+      <svg width="31" height="26" viewBox="0 0 31 26" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path d="M26.9067 25.4993C28.4263 23.6894 29.4982 21.5472 30.0358 19.246C30.5734 16.9448 30.5616 14.5493 30.0012 12.2535C29.4408 9.95778 28.3477 7.82624 26.8103 6.03152C25.2729 4.2368 23.3345 2.82935 21.152 1.92312C18.9695 1.01689 16.6043 0.637356 14.2478 0.815252C11.8914 0.993148 9.60991 1.72347 7.58819 2.94709C5.56648 4.17071 3.86134 5.85322 2.61082 7.85841C1.3603 9.86359 0.599568 12.1351 0.390208 14.489L6.84065 15.0627C6.95992 13.7217 7.39331 12.4276 8.10573 11.2853C8.81815 10.1429 9.78957 9.18438 10.9413 8.48729C12.0931 7.79019 13.3928 7.37413 14.7353 7.27278C16.0778 7.17144 17.4253 7.38765 18.6686 7.90393C19.912 8.42021 21.0163 9.22204 21.8922 10.2445C22.768 11.2669 23.3908 12.4813 23.71 13.7892C24.0292 15.0971 24.036 16.4618 23.7297 17.7727C23.4235 19.0837 22.8128 20.3042 21.9471 21.3353L26.9067 25.4993Z" fill="#EBEBFF"/>
+      </svg>
+    </div>
+    <div class="shape-10">
+      <svg width="20" height="21" viewBox="0 0 20 21" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <circle cx="10" cy="10.0956" r="10" fill="#6070FF"/>
+      </svg>
+    </div>
+    <div class="shape-11">
+      <svg width="68" height="39" viewBox="0 0 68 39" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <rect x="4.82422" y="-0.00012207" width="68.0631" height="12.1762" transform="rotate(23.34 4.82422 -0.00012207)" fill="#EBEBFF"/>
+      </svg>
+    </div>
+    <div class="shape-12">
+      <svg width="15" height="15" viewBox="0 0 15 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <circle cx="7.5" cy="7.49988" r="7" fill="white" stroke="#CBCDFF"/>
+      </svg>
+    </div>
+  </section>
```

Comment:
- Hey, let's improve the section structure to a better layout as this is basically all you need to do for this first section
```
<section class="intro">
    <h1>I'm Majek Glad to see you!</h1>
    <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
    <h3 class="links">LET'S CONNECT</h3>

<ul class="social-icons">
    <li><a href=""><img src="" alt=""></a></li>
    <li><a href=""><img src="" alt=""></a></li>
    <li><a href=""><img src="" alt=""></a></li>
    <li><a href=""><img src="" alt=""></a></li>
    <li><a href=""><img src="" alt=""></a></li>
  </ul>
  </section>
```
- From here we can use the CSS background property to add the background image, instead of hardcoding it using SVG. All you need to do is to download the complete image from the Figma design.
- For the social media icons, You can also add them completely as an image
- Using the correct styles from Figma will improve your design following the layout above.
- Please, let's improve the layout of the section to make our codebase more organized and maintainable.
_NOTE: This is just one out of million ways to achieve this_

Reply:
Fixed.